main.tf

```diff
@@ -0,0 +1,37 @@
+/**
+ * # AWS EKS Load Balancer Controller Terraform module
+ *
+ * A Terraform module to deploy the AWS Load Balancer Controller on Amazon EKS cluster.
+ *
+ * [![Terraform validate](https://github.com/lablabs/terraform-aws-eks-load-balancer-controller/actions/workflows/validate.yaml)
+ * [![pre-commit](https://github.com/lablabs/terraform-aws-eks-load-balancer-controller/actions/workflows/pre-commit.yml)
+ */
+locals {
+  addon = {
+    name = "aws-load-balancer-controller"
+
+    helm_chart_name    = "aws-load-balancer-controller"
```

Comment:
We can omit this as local.addon.name is used by default

Reply:
fixed.